x-pack/test/performance/tests/index.ts

```diff
@@ -11,6 +11,6 @@ export default function ({ loadTestFile }: FtrProviderContext) {
   describe('performance', function () {
     this.tags('ciGroup8');
 
-    loadTestFile(require.resolve('./reporting_dashboard'));
+    loadTestFile(require.resolve('./home'));
```

Comment:
Does it mean that `x-pack/test/performance/tests/reporting_dashboard.ts` test suit can be removed?

Reply:
Yes

Reply:
Should be able to delete that file as part of this PR